CorsixTH/Lua/app.lua

```diff
@@ -1443,6 +1443,27 @@ function App:checkInstallFolder()
   return true, #corrupt ~= 0 and corrupt or nil
 end
 
+function App:findSoundFont()
+  local data_dir = debug.getinfo(1, "S").source:sub(2, -12)
+
+  local possible_locations = {
+    self.config.soundfont or false,
```

Comment:
Code for the relevant config file areas affected will need to be added either now or in a future PR, ideally before 0.67.

Reply:
Preferably yes.